### DISCUSSION TITLE
Why is Id AutoGenerated anew on UPDATE with Relation using Symfony Doctrine? #9080

### DISCUSSION THREAD

**Discussion by dVVIIb** [doctrine/doctrine2#9129] [Support Questions]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | yes/no?
| ORM Version     | 2.9

#### Summary

Sorry, I think I misunderstood the version. I meant to say doctrine/doctrine-bundle ^2.4, but doctrine/orm ^2.9, which is the current stable?. Since I'm unable to reopen or comment, I'm reposting.

**Comment by bhushan** [comment]
You can use discussion section for asking such a queries .. also it would be nice if you provide more information about your question!!
